edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -736,6 +736,7 @@ impl ModuleRuntime for DockerModuleRuntime {
         // be emitted from Identity Service
         let provisioning = ProvisioningInfo {
             r#type: "ProvisioningType".into(),
+            //TODO: read from settings
```

Comment:
Since it's covered by the comment on line 735 anyway, and especially since line 741 is also already returning a fake value, I guess this comment isn't necessary.

Reply:
Fixed